Formula/rpm.rb

```diff
@@ -3,7 +3,7 @@ class Rpm < Formula
   homepage "https://rpm.org/"
   url "http://ftp.rpm.org/releases/rpm-4.15.x/rpm-4.15.1.tar.bz2"
   sha256 "ddef45f9601cd12042edfc9b6e37efcca32814e1e0f4bb8682d08144a3e2d230"
-  version_scheme 1
+  version_scheme 2
```

Comment:
Shouldn't the revision be bumped instead?

Reply:
To be honest I don't know what `version_scheme` is for. Thought it's some fancy revision-like function.

Reply:
In some sense it is. It's typically when upstream changes their versioning system and the new version might be seen by `brew` as an earlier version. For example, if a project switched from YYYY.MM.DD format to semver then we'll need to bump version_scheme. Basically the order of comparisons is `version_scheme`, `version`, `revision`.

`revision` is reset on version upgrades. `version_scheme` is never ever reset.